frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.tsx

```diff
@@ -43,6 +43,7 @@ export interface BaseBucketPickerPopoverProps {
   className?: string;
   classNames?: {
     root?: string;
+    triggerButton?: string;
```

Comment:
`root` seems to also cover the outer container (maybe duplicate class names being passed?), so I added a separate `triggerButton` class name which works quite well